paper/paper_quarto.html

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```

Comment:
A this an HTML version of it? (we could try to add it to the website)?

Reply:
yep, html version. Good idea - i'll try adding it to the docs!